Clients/GUI/Pages/SignUpWizzard.razor

```diff
@@ -0,0 +1,126 @@
+﻿@page "/signup"
+@using System.IO
+@using GUI.Components
+@using GUI.Scripts
+@using GUI.ViewModels
+@inject NavigationManager NavigationManager
+<EditForm Model="@userInput" OnValidSubmit="@TrySignUp">
+        <Wizard Id="Wizard">
+            <WizardStep Name="First Step">
+                <DataAnnotationsValidator />
+                <div class="form-group">
+                    <label for="Name">Name:</label>
+                    <InputText class="form-control col-3" @bind-Value="userInput.FirstName" placeholder="Your name" />
+                </div>
+                <div class="form-group">
+                    <label for="Surname">Surname:</label>
+                    <InputText class="form-control col-3" @bind-Value="userInput.LastName" placeholder="Your surname" />
+                </div>
+                <div class="form-group">
+                    <label for="Birthday">Birthday:</label>
+                    <InputDate class="form-control col-3" @bind-Value="userInput.Birthday" />
+                </div>
+                <div class="form-group">
+                    <label for="Email">Email</label>
+                    <InputText class="form-control col-3" @bind-Value="userInput.Email" placeholder="example@gmail.com" />
+                </div>
+                <div class="form-group">
+                    <label for="Password">Password</label>
+                    <InputText class="form-control col-3" type="password" @bind-Value="userInput.Password" placeholder="*****" />
+                </div>
+
+                <div class="form-group">
+                    <label for="ConfirmPassword">Confirm Password</label>
+                    <InputText class="form-control col-3" type="password" @bind-Value="userInput.ConfirmPassword" placeholder="*****" />
+                </div>
+                <br />
+
+                <ValidationSummary />
+                @if (isErrorDisplayed)
+                {
+                    <label style=" color: red;"> @errorMessage </label>
+                    <br />
+                }
+
+            </WizardStep>
+            <WizardStep Name="Second Step">
+                <div class="row">
+                    <div class="container m-5">
+                        <label>Send your avatar</label>
+                        <br />
+                        <InputFile OnChange="HandleSelection" />
+                        <br />
+                        @if (isErrorDisplayed)
+                        {
+                            <label style="color: red;"> @errorMessage </label>
+                            <br />
+                        }
+                        <ValidationSummary />
```

Comment:
Try to make it work better, wihout repeating errors, but if it won't be obvious how to do it, then we can leave it like that for a while

Reply:
fixed